lib/dehinter/font.py

```diff
@@ -14,12 +14,121 @@
 
 import array
 import os
+import pprint
+import sys
 from typing import Union
 
 from fontTools import ttLib  # type: ignore
 
 from dehinter.bitops import clear_bit_k, is_bit_k_set
 
+# instantiate pretty printer
+pp = pprint.PrettyPrinter(indent=4)
+
+
+# ========================================================
+# Core dehinting routine
+# ========================================================
+def dehint(tt,
+           keep_cvar=False,
+           keep_cvt=False,
+           keep_fpgm=False,
+           keep_gasp=False,
+           keep_glyf=False,
+           keep_hdmx=False,
+           keep_head=False,
+           keep_ltsh=False,
+           keep_maxp=False,
+           keep_prep=False,
+           keep_ttfa=False,
+           keep_vdmx=False,
+           verbose=True):
+    def report_actions(table, has_table):
+        if not has_table:
+            print("[-] Removed cvt table")
```

Comment:
```suggestion
            print(f"[-] Removed {table} table")
```

We'll need to fix this.  When I attempted to run the executable locally, I see the following stdout report:

```
[-] Removed cvt table
[-] Removed cvt table
[-] Removed cvt table
[-] Removed glyf table instruction bytecode from 2529 glyphs
[Δ] New gasp table values:
    {'gaspRange': {65535: 10}, 'tableTag': 'gasp', 'version': 1}
[Δ] New maxp table values:
    {   'maxComponentDepth': 6,
    'maxComponentElements': 5,
    'maxCompositeContours': 7,
    'maxCompositePoints': 106,
    'maxContours': 16,
    'maxFunctionDefs': 0,
    'maxInstructionDefs': 0,
    'maxPoints': 120,
    'maxSizeOfInstructions': 0,
    'maxStackElements': 0,
    'maxStorage': 0,
    'maxTwilightPoints': 0,
    'maxZones': 0,
    'numGlyphs': 2548,
    'tableTag': 'maxp',
    'tableVersion': 65536}

[+] Saved dehinted font as 'Inter-Regular-dehinted.ttf'

[*] File sizes:
    664.17KB (hinted)
    434.34KB (dehinted)
```

Note the "removed cvt table" in triplicate.  This should report the table names passed into the function.  cvt is currently hard coded into the string literal.

Reply:
good catch! This was a mistake. I'll fix it now.